CHANGELOG.md

```diff
@@ -24,6 +24,11 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * Added `Sequence.observed_chars` property for obtaining the set of observed characters in a sequence. ([#1075](https://github.com/biocore/scikit-bio/issues/1075))
 * Added `Sequence.frequencies` method for computing character frequencies in a sequence. ([#1074](https://github.com/biocore/scikit-bio/issues/1074))
 * Added experimental class-method ``Sequence.concat`` which will produce a new sequence from an iterable of existing sequences. Parameters control how positional metadata is propagated during a concatenation.
+* Added phylogenetic diversity metrics, including weighted UniFrac, unweighted UniFrac, and Faith's Phylogenetic Diversity. These are accessible as ``skbio.diversity.beta.unweighted_unifrac``, ``skbio.diversity.beta.weighted_unifrac``, and ````skbio.diversity.alpha.faith_pd``, respectively.
+* Addition of the function ``skbio.diversity.alpha_diversity`` to support applying an alpha diversity metric to multiple samples in one call.
+* Addition of the functions ``skbio.diversity.get_alpha_diversity_metrics`` and ``skbio.diversity.get_beta_diversity_metrics`` to support discovery of the alpha and beta diversity metrics implemented in scikit-bio.
+* ``TreeNode.to_array`` now supports replace ``nan`` branch lengths in the resulting branch length vector with the value provided as ``nan_length_value``.
+* Added ``skbio.util.assert_series_almost_equal`` function for comparing ``pd.DataFrame`` objects in unit tests.
```

Comment:
Keep this function package-private (importable from `skbio.util._testing.assert_series_almost_equal`) and remove changelog mention. (We'll be making `assert_data_frame_almost_equal` and `assert_ordination_results_equal` private soon, it was a mistake to make them public).